Update Docs to Allow Easier Copy and Paste

jvanderaa opened the pull request:
<!--
    Thank you for your interest in contributing to Nautobot! Please note
    that our contribution policy recommends that a feature request or bug
    report be opened for approval prior to filing a pull request. This
    helps avoid wasting time and effort on something that we might not
    be able to accept.

    Please indicate the relevant feature request or bug report below.
-->
# Closes: #2817 
# What's Changed
<!--
    Please include:
    - A summary of the proposed changes
    - A sectioned breakdown for larger features under ## subheadings
    - Screenshots, example payloads where relevant:
      - Before/After for bugfixes
      - Using a new feature
-->
In the mkdocs environment there is a copy code button. However, this button is not usable as is because the prompt is maintained within the code copy. This PR removes all instances of `$ ` indicating a prompt and adds example output where there was previously example output.

This allows for quick copy and paste throughout the docs.

A follow up PR once this is merged will also look at merging Ubuntu & RHEL into one document similar to how FastAPI references the differing platforms in their docs. So that way it can be a single threaded install guide.


# TODO
<!--
    Please feel free to update todos to keep track of your own notes for WIP PRs.
-->
- [x] Explanation of Change(s)
- [x] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/core/en/stable/development/#creating-changelog-fragments))
- [x] Attached Screenshots, Payload Example - Need to find this, no invoke command on how to show the docs page.
- [x] Unit, Integration Tests
- [x] Documentation Updates (when adding/changing features)
- [x] Example Plugin Updates (when adding/changing features)
- [x] Outline Remaining Work, Constraints from Design

glennmatthews commented on nautobot/docs/administration/nautobot-server.md:
```diff
@@ -7,23 +7,23 @@ Nautobot includes a command-line (CLI) management utility called `nautobot-serve
 !!! important
-    Since Nautobot is a Django application, there are a number of built-in management commands that will not be covered in this document. Please see the [official Django documentation on management commands](https://docs.djangoproject.com/en/stable/ref/django-admin/#available-commands) for more information.
+Since Nautobot is a Django application, there are a number of built-in management commands that will not be covered in this document. Please see the [official Django documentation on management commands](https://docs.djangoproject.com/en/stable/ref/django-admin/#available-commands) for more information.
 
 !!! important
-    Django does not recognize `nautobot-server`.  Anywhere `python manage.py` is mentioned, it is safe to replace with `nautobot-server`.
+Django does not recognize `nautobot-server`. Anywhere `python manage.py` is mentioned, it is safe to replace with `nautobot-server`.
```
Incorrect changes (autoformatter gone awry?) Please revert indentation to the correct 4-space value.

glennmatthews commented on nautobot/docs/administration/nautobot-server.md:
```diff
@@ -68,7 +72,7 @@ celery@worker1 v5.1.1 (sun-harmonics)
 
 !!! note
-    The internals of this command are built into Celery. Please see the [official Celery workers guide](https://docs.celeryq.dev/en/stable/userguide/workers.html) for more information.
+The internals of this command are built into Celery. Please see the [official Celery workers guide](https://docs.celeryq.dev/en/stable/userguide/workers.html) for more information.
```
revert

glennmatthews commented on nautobot/docs/administration/nautobot-server.md:
```diff
@@ -77,13 +81,17 @@ celery@worker1 v5.1.1 (sun-harmonics)
 
 !!! note
-    This is a built-in Django command. Please see the [official documentation on `collectstatic`](https://docs.djangoproject.com/en/stable/ref/django-admin/#collectstatic) for more information.
+This is a built-in Django command. Please see the [official documentation on `collectstatic`](https://docs.djangoproject.com/en/stable/ref/django-admin/#collectstatic) for more information.
```
revert

glennmatthews commented on nautobot/docs/administration/nautobot-server.md:
```diff
@@ -101,7 +114,7 @@ Superuser created successfully.
 
 !!! note
-    This is a built-in Django command. Please see the [official documentation on `createsuperuser`](https://docs.djangoproject.com/en/stable/ref/django-admin/#createsuperuser) for more information.
+This is a built-in Django command. Please see the [official documentation on `createsuperuser`](https://docs.djangoproject.com/en/stable/ref/django-admin/#createsuperuser) for more information.
```
revert

glennmatthews commented on nautobot/docs/administration/nautobot-server.md:
```diff
@@ -110,10 +123,15 @@ Superuser created successfully.
 
 !!! danger
-    This is an advanced feature that gives you direct access to run raw SQL queries. Use this very cautiously as you can cause irreparable damage to your Nautobot installation.
+This is an advanced feature that gives you direct access to run raw SQL queries. Use this very cautiously as you can cause irreparable damage to your Nautobot installation.
```
revert

glennmatthews commented on nautobot/docs/administration/nautobot-server.md:
```diff
@@ -125,16 +143,14 @@ nautobot=> \q
 
 !!! note
-    This is a built-in Django command. Please see the [official documentation on `dbshell`](https://docs.djangoproject.com/en/stable/ref/django-admin/#dbshell) for more information.
+This is a built-in Django command. Please see the [official documentation on `dbshell`](https://docs.djangoproject.com/en/stable/ref/django-admin/#dbshell) for more information.
```
revert

glennmatthews commented on nautobot/docs/administration/nautobot-server.md:
````diff
@@ -125,16 +143,14 @@ nautobot=> \q
 ```
 
 !!! note
-    This is a built-in Django command. Please see the [official documentation on `dbshell`](https://docs.djangoproject.com/en/stable/ref/django-admin/#dbshell) for more information.
+This is a built-in Django command. Please see the [official documentation on `dbshell`](https://docs.djangoproject.com/en/stable/ref/django-admin/#dbshell) for more information.
 
 ### `dumpdata`
 
-+/- 1.3.0
-    - `extras.job` should now be included in the dump (removed `--exclude extras.job` from the example usage)
-    - `django_rq` should now be excluded from the dump (added `--exclude django_rq` to the example usage)
++/- 1.3.0 - `extras.job` should now be included in the dump (removed `--exclude extras.job` from the example usage) - `django_rq` should now be excluded from the dump (added `--exclude django_rq` to the example usage)
````
revert all of this please

glennmatthews commented on nautobot/docs/administration/nautobot-server.md:
```diff
@@ -190,13 +211,18 @@ Populate the database with various data as a baseline for testing (automated or
 
 !!! danger
-    Running this command with the `--flush` argument will clear all existing data in your database. You have been warned.
+Running this command with the `--flush` argument will clear all existing data in your database. You have been warned.
```
revert

glennmatthews commented on nautobot/docs/administration/nautobot-server.md:
```diff
@@ -249,13 +285,13 @@ For more information on configuring Nautobot for the first time or on more advan
 
 !!! note
-    This is a built-in management command provided by the [Cacheops plugin](https://github.com/Suor/django-cacheops) Nautobot for caching. Please see the official [Cacheops documentation on `invalidate`](https://github.com/Suor/django-cacheops#invalidation) for more information.
+This is a built-in management command provided by the [Cacheops plugin](https://github.com/Suor/django-cacheops) Nautobot for caching. Please see the official [Cacheops documentation on `invalidate`](https://github.com/Suor/django-cacheops#invalidation) for more information.
```
revert

glennmatthews commented on nautobot/docs/administration/nautobot-server.md:
```diff
@@ -285,7 +326,7 @@ Running migrations:
 
 !!! note
-    This is a built-in Django command. Please see the [official documentation on `migrate`](https://docs.djangoproject.com/en/stable/ref/django-admin/#migrate) for more information.
+This is a built-in Django command. Please see the [official documentation on `migrate`](https://docs.djangoproject.com/en/stable/ref/django-admin/#migrate) for more information.
```
revert

glennmatthews commented on nautobot/docs/administration/nautobot-server.md:
```diff
@@ -294,10 +335,15 @@ Running migrations:
 
 !!! danger
-    This is an advanced feature that gives you direct access to the Django database models. Use this very cautiously as you can cause irreparable damage to your Nautobot installation.
+This is an advanced feature that gives you direct access to the Django database models. Use this very cautiously as you can cause irreparable damage to your Nautobot installation.
```
revert

glennmatthews commented on nautobot/docs/administration/nautobot-server.md:
```diff
@@ -322,7 +368,7 @@ This will run the following management commands with default settings, in order:
 
 !!! note
-    Commands listed here that are not covered in this document here are Django built-in commands.
+Commands listed here that are not covered in this document here are Django built-in commands.
```
revert

glennmatthews commented on nautobot/docs/administration/nautobot-server.md:
```diff
@@ -450,10 +511,15 @@ Please see the [guide on Jobs](../additional-features/jobs.md) for more informat
 
 !!! note
-    uWSGI offers an overwhelming amount of command-line arguments that could not possibly be covered here. Please see the [official uWSGI Options guide](https://uwsgi-docs.readthedocs.io/en/latest/Options.html) for more information.
+uWSGI offers an overwhelming amount of command-line arguments that could not possibly be covered here. Please see the [official uWSGI Options guide](https://uwsgi-docs.readthedocs.io/en/latest/Options.html) for more information.
```
revert

glennmatthews commented on nautobot/docs/administration/nautobot-server.md:
```diff
@@ -536,7 +607,7 @@ Finished.
 
 !!! note
-    This command is safe to run at any time. If it does detect any changes, it will exit cleanly.
+This command is safe to run at any time. If it does detect any changes, it will exit cleanly.
```
revert

glennmatthews commented on nautobot/docs/administration/nautobot-shell.md:
```diff
@@ -29,7 +34,7 @@ DCIM:
 
 !!! warning
-    The Nautobot shell affords direct access to Nautobot data and function with very little validation in place. As such, it is crucial to ensure that only authorized, knowledgeable users are ever granted access to it. Never perform any action in the management shell without having a full backup in place.
+The Nautobot shell affords direct access to Nautobot data and function with very little validation in place. As such, it is crucial to ensure that only authorized, knowledgeable users are ever granted access to it. Never perform any action in the management shell without having a full backup in place.
```
revert

glennmatthews commented on nautobot/docs/administration/nautobot-shell.md:
```diff
@@ -106,7 +111,7 @@ This approach can span multiple levels of relations. For example, the following
 
 !!! note
-    While the above query is functional, it's not very efficient. There are ways to optimize such requests, however they are out of scope for this document. For more information, see the [Django queryset method reference](https://docs.djangoproject.com/en/stable/ref/models/querysets/) documentation.
+While the above query is functional, it's not very efficient. There are ways to optimize such requests, however they are out of scope for this document. For more information, see the [Django queryset method reference](https://docs.djangoproject.com/en/stable/ref/models/querysets/) documentation.
```
revert

glennmatthews commented on nautobot/docs/administration/nautobot-shell.md:
````diff
@@ -144,7 +149,7 @@ To return the inverse of a filtered queryset, use `exclude()` instead of `filter
 ```
 
 !!! info
-    The examples above are intended only to provide a cursory introduction to queryset filtering. For an exhaustive list of the available filters, please consult the [Django queryset API documentation](https://docs.djangoproject.com/en/stable/ref/models/querysets/).
````
revert

glennmatthews commented on nautobot/docs/administration/nautobot-shell.md:
````diff
@@ -175,10 +180,10 @@ To modify an existing object, we retrieve it, update the desired field(s), and c
 ```
 
 !!! warning
-    It is recommended to make use of the `validated_save()` convenience method which exists on all core models. While the Django `save()` method still exists, the `validated_save()` method saves the instance data but first enforces model validation logic. Simply calling `save()` on the model instance **does not** enforce validation automatically and may lead to bad data. See the development [best ...
+It is recommended to make use of the `validated_save()` convenience method which exists on all core models. While the Django `save()` method still exists, the `validated_save()` method saves the instance data but first enforces model validation logic. Simply calling `save()` on the model instance **does not** enforce validation automatically and may lead to bad data. See the development [best pra ...
 
 !!! warning
-    The Django ORM provides methods to create/edit many objects at once, namely `bulk_create()` and `update()`. These are best avoided in most cases as they bypass a model's built-in validation and can easily lead to database corruption if not used carefully.
+The Django ORM provides methods to create/edit many objects at once, namely `bulk_create()` and `update()`. These are best avoided in most cases as they bypass a model's built-in validation and can easily lead to database corruption if not used carefully.
````
revert

glennmatthews commented on nautobot/docs/administration/nautobot-shell.md:
```diff
@@ -203,7 +208,7 @@ To delete multiple objects at once, call `delete()` on a filtered queryset. It's
 
 !!! warning
-    Deletions are immediate and irreversible. Always consider the impact of deleting objects carefully before calling `delete()` on an instance or queryset.
+Deletions are immediate and irreversible. Always consider the impact of deleting objects carefully before calling `delete()` on an instance or queryset.
```
revert

glennmatthews commented on nautobot/docs/configuration/authentication/ldap.md:
```diff
@@ -9,42 +9,45 @@ This guide explains how to implement LDAP authentication using an external serve
 
 !!! warning
-    This and all remaining steps in this document should all be performed as the `nautobot` user!
+This and all remaining steps in this document should all be performed as the `nautobot` user!
```
revert

glennmatthews commented on nautobot/docs/configuration/authentication/ldap.md:
```diff
@@ -9,42 +9,45 @@ This guide explains how to implement LDAP authentication using an external serve
 !!! note
-    It is critical that you include the `ObjectPermissionsBackend` provided by
-    Nautobot after the `LDAPBackend` so that object-level permissions features can work properly.
+It is critical that you include the `ObjectPermissionsBackend` provided by
+Nautobot after the `LDAPBackend` so that object-level permissions features can work properly.
```
revert

glennmatthews commented on nautobot/docs/configuration/authentication/ldap.md:
```diff
@@ -58,7 +61,7 @@ AUTHENTICATION_BACKENDS = [
 
 !!! info
-    When using Windows Server 2012 you may wish to use the [Global Catalog](https://docs.microsoft.com/en-us/windows/win32/ad/global-catalog) by specifying a port on `AUTH_LDAP_SERVER_URI`. Use `3269` for secure (`ldaps://`), or `3268` for non-secure.
+When using Windows Server 2012 you may wish to use the [Global Catalog](https://docs.microsoft.com/en-us/windows/win32/ad/global-catalog) by specifying a port on `AUTH_LDAP_SERVER_URI`. Use `3269` for secure (`ldaps://`), or `3268` for non-secure.
```
revert

glennmatthews commented on nautobot/docs/configuration/authentication/ldap.md:
```diff
@@ -123,7 +126,7 @@ Additional ldap connection options can be found in the [python-ldap documentatio
 
 !!! info
-    When using Windows Server 2012, `AUTH_LDAP_USER_DN_TEMPLATE` should be set to None.
+When using Windows Server 2012, `AUTH_LDAP_USER_DN_TEMPLATE` should be set to None.
```
revert

glennmatthews commented on nautobot/docs/configuration/authentication/ldap.md:
```diff
@@ -174,7 +177,7 @@ if AUTH_LDAP_USER_SEARCH_DN != "":
 
 !!! info
-    When using Microsoft Active Directory, support for nested groups can be activated by using `NestedGroupOfNamesType()` instead of `GroupOfNamesType()` for `AUTH_LDAP_GROUP_TYPE`. You will also need to modify the import line to use `NestedGroupOfNamesType` instead of `GroupOfNamesType` .
+When using Microsoft Active Directory, support for nested groups can be activated by using `NestedGroupOfNamesType()` instead of `GroupOfNamesType()` for `AUTH_LDAP_GROUP_TYPE`. You will also need to modify the import line to use `NestedGroupOfNamesType` instead of `GroupOfNamesType` .
```
revert

glennmatthews commented on nautobot/docs/configuration/authentication/ldap.md:
```diff
@@ -147,7 +150,7 @@ AUTH_LDAP_USER_ATTR_MAP = {
 #### Searching in Multiple LDAP Groups
 
-Define the user-groups in your environment, such as a *.env file (delimiter `';'`):
+Define the user-groups in your environment, such as a \*.env file (delimiter `';'`):
```
Perhaps:

```suggestion
Define the user-groups in your environment, such as a `*.env` file (delimiter `';'`):
```

glennmatthews commented on nautobot/docs/configuration/authentication/ldap.md:
```diff
@@ -203,18 +206,18 @@ AUTH_LDAP_CACHE_TIMEOUT = 3600
 
 !!! warning
-    Authentication will fail if the groups (the distinguished names) do not exist in the LDAP directory.
+Authentication will fail if the groups (the distinguished names) do not exist in the LDAP directory.
```
revert

glennmatthews commented on nautobot/docs/configuration/authentication/ldap.md:
```diff
@@ -250,7 +253,7 @@ AUTH_LDAP_SECONDARY_SERVER_URI = "ldap://secondary-ad.example.com"
 
 !!! info
-    In this example the default LDAPBackend was still used as the first LDAP server, which utilized the `AUTH_LDAP_*` environment variables. It is also possible to remove the default backend and create multiple custom backends instead to normalize the environment variable naming scheme.
+In this example the default LDAPBackend was still used as the first LDAP server, which utilized the `AUTH_LDAP_*` environment variables. It is also possible to remove the default backend and create multiple custom backends instead to normalize the environment variable naming scheme.
```
revert

glennmatthews commented on nautobot/docs/configuration/index.md:
```diff
@@ -9,22 +9,30 @@ An initial configuration can be created by executing `nautobot-server init`. Thi
 
 !!! tip
-    The [Nautobot Installation Docs](../installation/nautobot.md#choose-your-nautobot_root) example sets `NAUTOBOT_ROOT` to `/opt/nautobot`, so `nautobot_config.py` would be found at `/opt/nautobot/nautobot_config.py`.
+The [Nautobot Installation Docs](../installation/nautobot.md#choose-your-nautobot_root) example sets `NAUTOBOT_ROOT` to `/opt/nautobot`, so `nautobot_config.py` would be found at `/opt/nautobot/nautobot_config.py`.
```
revert

glennmatthews commented on nautobot/docs/configuration/index.md:
```diff
@@ -9,22 +9,30 @@ An initial configuration can be created by executing `nautobot-server init`. Thi
 
 !!! note
-    Throughout the documentation, the configuration file will be referred to by name as `nautobot_config.py`. If you use a custom file name, you must use that instead.
+Throughout the documentation, the configuration file will be referred to by name as `nautobot_config.py`. If you use a custom file name, you must use that instead.
```
revert

glennmatthews commented on nautobot/docs/configuration/index.md:
````diff
@@ -56,16 +64,16 @@ By default, Nautobot will always read or store files in `~/.nautobot` to allow f
 
 ```no-highlight
-$ export NAUTOBOT_ROOT=/opt/nautobot
+export NAUTOBOT_ROOT=/opt/nautobot
 ```
 
 This setting is also used in the [Nautobot deployment guide](../installation/nautobot.md) to make the `nautobot-server` command easier to find and use.
 
 !!! note
-    The `--config` argument and the `NAUTOBOT_CONFIG` environment variable will always take precedence over `NAUTOBOT_ROOT` for the purpose of telling Nautobot where your `nautobot_config.py` can be found.
+The `--config` argument and the `NAUTOBOT_CONFIG` environment variable will always take precedence over `NAUTOBOT_ROOT` for the purpose of telling Nautobot where your `nautobot_config.py` can be found.
 
 !!! warning
-    Do not override `NAUTOBOT_ROOT` in your `nautobot_config.py`. It will not work as expected. If you need to customize this setting, please always set the `NAUTOBOT_ROOT` environment variable.
+Do not override `NAUTOBOT_ROOT` in your `nautobot_config.py`. It will not work as expected. If you need to customize this setting, please always set the `NAUTOBOT_ROOT` environment variable.
````
revert

glennmatthews commented on nautobot/docs/configuration/index.md:
```diff
@@ -120,15 +133,15 @@ Inspect the `SETTINGS_PATH` variable. Does it match the configuration you're exp
 
 !!! warning
-    Be cautious not to confuse extra applications with Nautobot plugins which are installed using the [`PLUGINS`](optional-settings.md#plugins) setting. They are similar, but distinctly different!
+Be cautious not to confuse extra applications with Nautobot plugins which are installed using the [`PLUGINS`](optional-settings.md#plugins) setting. They are similar, but distinctly different!
```
revert

glennmatthews commented on nautobot/docs/configuration/required-settings.md:
```diff
@@ -373,23 +373,33 @@ Please note that this key is **not** used directly for hashing user passwords or
 
 !!! note
-    A unique `SECRET_KEY` is generated for you automatically when you use `nautobot-server init` to create a new `nautobot_config.py`.
+A unique `SECRET_KEY` is generated for you automatically when you use `nautobot-server init` to create a new `nautobot_config.py`.
```
please revert all changes here and above to this file.

glennmatthews commented on nautobot/docs/configuration/required-settings.md:
```diff
@@ -373,23 +373,33 @@ Please note that this key is **not** used directly for hashing user passwords or
 
 !!! warning
-    In the case of a highly available installation with multiple web servers, `SECRET_KEY` must be identical among all servers in order to maintain a persistent user session state.
+In the case of a highly available installation with multiple web servers, `SECRET_KEY` must be identical among all servers in order to maintain a persistent user session state.
```
revert

glennmatthews commented on nautobot/docs/development/docker-compose-advanced-use-cases.md:
```diff
@@ -148,7 +148,7 @@ After opening the project directory in VS Code in a supported environment, you w
 
 !!! note
-    You can run tests with `nautobot-server --config=nautobot/core/tests/nautobot_config.py test nautobot` while inside the Container.
+You can run tests with `nautobot-server --config=nautobot/core/tests/nautobot_config.py test nautobot` while inside the Container.
```
revert

glennmatthews commented on nautobot/docs/development/extending-models.md:
```diff
@@ -7,17 +7,17 @@ Below is a list of tasks to consider when adding a new field to a core model.
 
 !!! warning
-    Assert that you have installed Nautobot in your development environment using `poetry install` so that changes you make to migrations will apply to the source tree!
+Assert that you have installed Nautobot in your development environment using `poetry install` so that changes you make to migrations will apply to the source tree!
```
revert

glennmatthews commented on nautobot/docs/development/getting-started.md:
```diff
@@ -82,31 +113,31 @@ Before you make any changes, always create a new branch. Again, for bug fixes an
 
 !!! warning
-    If you do not do this, you run the risk of having merge conflicts in your branch, and that's never fun to deal with. Trust us on this one.
+If you do not do this, you run the risk of having merge conflicts in your branch, and that's never fun to deal with. Trust us on this one.
```
revert

glennmatthews commented on nautobot/docs/development/getting-started.md:
```diff
@@ -230,11 +266,11 @@ There are a few things you'll need:
 
 !!! danger
-    Always utilize this documented method to install Poetry for use when developing Nautobot.
+Always utilize this documented method to install Poetry for use when developing Nautobot.
```
revert

glennmatthews commented on nautobot/docs/development/getting-started.md:
```diff
@@ -247,41 +283,51 @@ For detailed installation instructions, please see the [official Poetry installa
 
 !!! hint
-    If you are doing development or testing using MySQL, you may quickly install the `mysqlclient` library along with Nautobot by running `poetry install --extras mysql`.
+If you are doing development or testing using MySQL, you may quickly install the `mysqlclient` library along with Nautobot by running `poetry install --extras mysql`.
```
revert

glennmatthews commented on nautobot/docs/development/getting-started.md:
````diff
@@ -315,18 +376,18 @@ Collecting ipython
 It may not always be convenient to enter into the virtual shell just to run programs. You may also execute a given command ad hoc within the project's virtual shell by using `poetry run`:
 
 ```no-highlight
-$ poetry run mkdocs serve
+poetry run mkdocs serve
 ```
 
 Check out the [Poetry usage guide](https://python-poetry.org/docs/basic-usage/) for more tips.
 
 #### Configuring Nautobot
 
 !!! note
-    Unless otherwise noted, all following commands should be executed inside the virtualenv.
+Unless otherwise noted, all following commands should be executed inside the virtualenv.
 
 !!! hint
-    Use `poetry shell` to enter the virtualenv.
+Use `poetry shell` to enter the virtualenv.
````
revert

glennmatthews commented on nautobot/docs/development/getting-started.md:
```diff
@@ -369,27 +435,32 @@ Below are common commands for working your development environment.
 
 !!! danger
-    **DO NOT USE THIS SERVER IN A PRODUCTION SETTING.** The development server and watchdog is for development and testing purposes only. It is neither performant nor secure enough for production use.
+**DO NOT USE THIS SERVER IN A PRODUCTION SETTING.** The development server and watchdog is for development and testing purposes only. It is neither performant nor secure enough for production use.
```
revert

glennmatthews commented on nautobot/docs/development/getting-started.md:
```diff
@@ -400,7 +471,7 @@ Quit the server with CONTROL-C.
 
 !!! warning
-    Do not use `poetry run nautobot-server runserver` as it will crash unless you also pass the `--noreload` flag, which somewhat defeats the purpose of using the development server. It is recommended to use `nautobot-server runserver` from within an active virtualenv (e.g. `poetry shell`). This is a [known issue with Django and Poetry](https://github.com/python-poetry/poetry/issues/2435).
+Do not use `poetry run nautobot-server runserver` as it will crash unless you also pass the `--noreload` flag, which somewhat defeats the purpose of using the development server. It is recommended to use `nautobot-server runserver` from within an active virtualenv (e.g. `poetry shell`). This is a [known issue with Django and Poetry](https://github.com/python-poetry/poetry/issues/2435).
```
revert

glennmatthews commented on nautobot/docs/development/getting-started.md:
```diff
@@ -601,17 +687,17 @@ You can also add text to any of these admonitions for further clarity, for examp
 will render as:
 
 +++ 1.4.3
-    The custom `mkdocs` plugin was added.
+The custom `mkdocs` plugin was added.
 
 !!! caution
-    While you *can* use the `version-added` / `version-changed` / `version-removed` admonitions directly to add a custom title to a specific admonition, in general, you should use the macros for consistency across the documentation.
+While you _can_ use the `version-added` / `version-changed` / `version-removed` admonitions directly to add a custom title to a specific admonition, in general, you should use the macros for consistency across the documentation.
```
revert

glennmatthews commented on nautobot/docs/development/getting-started.md:
```diff
@@ -620,25 +706,25 @@ Documentation is written in Markdown. If you need to add additional pages or sec
 
 !!! note
-    Remember, pull requests are entertained only for **accepted** issues. If an issue you want to work on hasn't been approved by a maintainer yet, it's best to avoid risking your time and effort on a change that might not be accepted.
+Remember, pull requests are entertained only for **accepted** issues. If an issue you want to work on hasn't been approved by a maintainer yet, it's best to avoid risking your time and effort on a change that might not be accepted.
```
revert

glennmatthews commented on nautobot/docs/development/release-checklist.md:
```diff
@@ -28,7 +28,7 @@ The other file is `poetry.lock`, which is managed by Poetry and contains package
 
 !!! warning
-    You must never directly edit this file. You will use `poetry update` commands to manage it.
+You must never directly edit this file. You will use `poetry update` commands to manage it.
```
revert

glennmatthews commented on nautobot/docs/development/release-checklist.md:
```diff
@@ -41,17 +41,17 @@ Every minor version release should refresh `poetry.lock`, so that it lists the m
 
 !!! hint
-    You may use `poetry update --dry-run` to have Poetry automatically tell you what package updates are available and the versions it would upgrade.
+You may use `poetry update --dry-run` to have Poetry automatically tell you what package updates are available and the versions it would upgrade.
```
revert

glennmatthews commented on nautobot/docs/development/release-checklist.md:
```diff
@@ -61,10 +61,10 @@ Add the release notes (`docs/release-notes/X.Y.md`) to the table of contents wit
 
 !!! tip
-    Fire up `mkdocs serve` in your development environment to start the documentation server! This allows you to view the documentation locally and automatically rebuilds the documents as you make changes.
+Fire up `mkdocs serve` in your development environment to start the documentation server! This allows you to view the documentation locally and automatically rebuilds the documents as you make changes.
```
revert

glennmatthews commented on nautobot/docs/development/release-checklist.md:
```diff
@@ -136,7 +161,7 @@ Check the git diff to verify the changes are correct (`git diff --cached`).
 
 !!! important
-    The changelog must adhere to the [Keep a Changelog](https://keepachangelog.com/) style guide.
+The changelog must adhere to the [Keep a Changelog](https://keepachangelog.com/) style guide.
```
revert

glennmatthews commented on nautobot/docs/development/release-checklist.md:
```diff
@@ -200,7 +225,7 @@ nautobot:
 
 !!! warning
-    You should *not* include `docker-compose.dev.yml` in this test scenario!
+You should _not_ include `docker-compose.dev.yml` in this test scenario!
```
revert

glennmatthews commented on nautobot/docs/docker/index.md:
```diff
@@ -98,18 +98,18 @@ Enables creation of a super user specified by [`NAUTOBOT_SUPERUSER_NAME`](#nauto
 
 !!! note
-    Setting this value to anything other than "false" (case-insensitive) will prevent migrations from occurring.
+Setting this value to anything other than "false" (case-insensitive) will prevent migrations from occurring.
```
revert

glennmatthews commented on nautobot/docs/installation/centos.md:
````diff
@@ -131,21 +143,26 @@ sudo dnf install -y gcc mysql-server mysql-devel
 Start the service and enable it to run at system startup:
 
 ```no-highlight
-$ sudo systemctl enable --now mysql
+sudo systemctl enable --now mysql
 ```
 
 #### Create a MySQL Database
 
 At a minimum, we need to create a database for Nautobot and assign it a username and password for authentication. This is done with the following commands.
 
 !!! note
-    Replace `localhost` below with your database server if using a remote database.
+Replace `localhost` below with your database server if using a remote database.
 
 !!! danger
-    **Do not use the password from the example.** Choose a strong, random password to ensure secure database authentication for your Nautobot installation.
+**Do not use the password from the example.** Choose a strong, random password to ensure secure database authentication for your Nautobot installation.
````
revert

glennmatthews commented on nautobot/docs/installation/http-server.md:
```diff
@@ -10,19 +10,19 @@ provider, obtain one for free from [Let's Encrypt](https://letsencrypt.org/getti
 !!! info
-    Some Linux installations, including CentOS, have changed the location for SSL certificates from `/etc/ssl/` to `/etc/pki/tls/`. The
-    command below may need to be changed to reflect the certificate location.
+Some Linux installations, including CentOS, have changed the location for SSL certificates from `/etc/ssl/` to `/etc/pki/tls/`. The
+command below may need to be changed to reflect the certificate location.
```
revert

glennmatthews commented on nautobot/docs/installation/http-server.md:
```diff
@@ -32,7 +32,7 @@ $ sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 \
 
 !!! warning
-    The following steps must be performed with root permissions.
+The following steps must be performed with root permissions.
```
revert

glennmatthews commented on nautobot/docs/installation/http-server.md:
```diff
@@ -115,47 +115,47 @@ To enable the Nautobot site, you'll need to delete `/etc/nginx/sites-enabled/def
 
 !!! info
-    If the restart fails, and you changed the default key location, check to make sure the `nautobot.conf` file you pasted has the updated key location. For example, CentOS requires keys to be in `/etc/pki/tls/` instead of `/etc/ssl/`.
+If the restart fails, and you changed the default key location, check to make sure the `nautobot.conf` file you pasted has the updated key location. For example, CentOS requires keys to be in `/etc/pki/tls/` instead of `/etc/ssl/`.
```
revert

glennmatthews commented on nautobot/docs/installation/nautobot.md:
```diff
@@ -4,16 +4,16 @@ This section of the documentation discusses installing and configuring the Nauto
 
 !!! important
-    Your database server and Redis must have been successfully installed before continuing with deployment steps. If you haven't done that yet, please visit the guide on [Installing Nautobot Dependencies](index.md#installing-nautobot-dependencies)
+Your database server and Redis must have been successfully installed before continuing with deployment steps. If you haven't done that yet, please visit the guide on [Installing Nautobot Dependencies](index.md#installing-nautobot-dependencies)
```
revert

glennmatthews commented on nautobot/docs/installation/nautobot.md:
```diff
@@ -22,7 +22,7 @@ You need to select a directory path where everything related to Nautobot will be
 
 !!! tip
-    We have selected `/opt/nautobot`, but you may use any directory you choose.
+We have selected `/opt/nautobot`, but you may use any directory you choose.
```
revert

glennmatthews commented on nautobot/docs/installation/nautobot.md:
```diff
@@ -33,26 +33,26 @@ Create a system user account named `nautobot`. This user will own all of the Nau
 
 !!! note
-    Instead of deliberately requiring you to activate/deactivate the virtualenv, we are emphasizing on relying on the `$PATH` to access programs installed within it. We find this to be much more intuitive and natural when working with Nautobot in this way.
+Instead of deliberately requiring you to activate/deactivate the virtualenv, we are emphasizing on relying on the `$PATH` to access programs installed within it. We find this to be much more intuitive and natural when working with Nautobot in this way.
```
revert

glennmatthews commented on nautobot/docs/installation/nautobot.md:
```diff
@@ -64,26 +64,31 @@ We need to set the `NAUTOBOT_ROOT` environment variable for the `nautobot` user
 
 !!! warning
-    Unless explicitly stated, all remaining steps requiring the use of `pip3` or `nautobot-server` in this document should be performed as the `nautobot` user!
+Unless explicitly stated, all remaining steps requiring the use of `pip3` or `nautobot-server` in this document should be performed as the `nautobot` user!
```
revert

glennmatthews commented on nautobot/docs/installation/nautobot.md:
```diff
@@ -126,29 +146,35 @@ Before we install anything into the virtualenv, we want to make sure that Pip is
 
 !!! hint
-    If you are using MySQL as your database backend, use `pip3 install "nautobot[mysql]"` to install Nautobot and the `mysqlclient` library together!
+If you are using MySQL as your database backend, use `pip3 install "nautobot[mysql]"` to install Nautobot and the `mysqlclient` library together!
```
revert

glennmatthews commented on nautobot/docs/installation/nautobot.md:
```diff
@@ -126,29 +146,35 @@ Before we install anything into the virtualenv, we want to make sure that Pip is
 
 !!! warning
-    If you're using a MySQL database, Nautobot **will not work** without this client library. You cannot skip this step.
+If you're using a MySQL database, Nautobot **will not work** without this client library. You cannot skip this step.
```
revert

glennmatthews commented on nautobot/docs/installation/nautobot.md:
````diff
@@ -172,25 +198,25 @@ Initialize a new configuration by running `nautobot-server init`. You may specif
 However, because we've set the `NAUTOBOT_ROOT`, this command will automatically create a new `nautobot_config.py` at the default location based on this at `$NAUTOBOT_ROOT/nautobot_config.py`:
 
 ```no-highlight
-$ nautobot-server init
+nautobot-server init
 Configuration file created at '/opt/nautobot/nautobot_config.py'
 ```
 
 ### Required Settings
 
-Your `nautobot_config.py` provides sane defaults for all of the configuration settings. You will inevitably need to update the settings for your environment, most notably the [`DATABASES`](../configuration/required-settings.md#databases) setting.  If you do not wish to modify the config, by default, many of these configuration settings can also be specified by environment variables. Please see [R ...
+Your `nautobot_config.py` provides sane defaults for all of the configuration settings. You will inevitably need to update the settings for your environment, most notably the [`DATABASES`](../configuration/required-settings.md#databases) setting. If you do not wish to modify the config, by default, many of these configuration settings can also be specified by environment variables. Please see [Re ...
 
 Edit `$NAUTOBOT_ROOT/nautobot_config.py`, and head over to the documentation on [Required Settings](../configuration/required-settings.md) to tweak your required settings. At a minimum, you'll need to update the following settings:
 
-* [`ALLOWED_HOSTS`](../configuration/required-settings.md#allowed_hosts): You must set this value. This can be set to `["*"]` for a quick start, but this value is not suitable for production deployment.
-* [`DATABASES`](../configuration/required-settings.md#databases): Database connection parameters. If you installed your database server on the same system as Nautobot, you'll need to update the `USER` and `PASSWORD` fields here. If you are using MySQL, you'll also need to update the `ENGINE` field, changing the default database driver suffix from `django.db.backends.postgresql` to `django.db.back ...
-* **Redis settings**: Redis configuration requires multiple settings including [`CACHEOPS_REDIS`](../configuration/required-settings.md#cacheops_redis) and [`RQ_QUEUES`](../configuration/required-settings.md#rq_queues), if different from the defaults. If you installed Redis on the same system as Nautobot, you do not need to change these settings.
+- [`ALLOWED_HOSTS`](../configuration/required-settings.md#allowed_hosts): You must set this value. This can be set to `["*"]` for a quick start, but this value is not suitable for production deployment.
+- [`DATABASES`](../configuration/required-settings.md#databases): Database connection parameters. If you installed your database server on the same system as Nautobot, you'll need to update the `USER` and `PASSWORD` fields here. If you are using MySQL, you'll also need to update the `ENGINE` field, changing the default database driver suffix from `django.db.backends.postgresql` to `django.db.back ...
+- **Redis settings**: Redis configuration requires multiple settings including [`CACHEOPS_REDIS`](../configuration/required-settings.md#cacheops_redis) and [`RQ_QUEUES`](../configuration/required-settings.md#rq_queues), if different from the defaults. If you installed Redis on the same system as Nautobot, you do not need to change these settings.
 
 !!! important
-    You absolutely must update your required settings in your `nautobot_config.py` or Nautobot will not work.
+You absolutely must update your required settings in your `nautobot_config.py` or Nautobot will not work.
 
 !!! warning
-    If you are using MySQL as your database backend, you **must also update** the database `ENGINE` setting to `django.db.backends.mysql`.
+If you are using MySQL as your database backend, you **must also update** the database `ENGINE` setting to `django.db.backends.mysql`.
````
revert

glennmatthews commented on nautobot/docs/installation/nautobot.md:
```diff
@@ -225,51 +251,51 @@ By default, Nautobot will use the local filesystem to store uploaded files. To u
 
 !!! note
-    If you did not create a `local_requirements.txt` above, please skip this step.
+If you did not create a `local_requirements.txt` above, please skip this step.
```
revert

glennmatthews commented on nautobot/docs/installation/nautobot.md:
```diff
@@ -279,10 +305,10 @@ Nautobot leverages Django's built-in [system check framework](https://docs.djang
 
 !!! hint
-    Get into the habit of running checks before deployments!
+Get into the habit of running checks before deployments!
```
revert

glennmatthews commented on nautobot/docs/installation/nautobot.md:
````diff
@@ -291,19 +317,19 @@ At this point, we should be able to run Nautobot's development server for testin
 development instance:
 
 ```no-highlight
-$ nautobot-server runserver 0.0.0.0:8080 --insecure
+nautobot-server runserver 0.0.0.0:8080 --insecure
 ```
 
 Next, connect to the name or IP of the server (as defined in `ALLOWED_HOSTS`) on port 8080; for example, <http://127.0.0.1:8080/>. You should be greeted with the Nautobot home page.
 
 !!! danger
-    **DO NOT USE THIS SERVER IN A PRODUCTION SETTING.** The development server is for development and testing purposes only. It is neither performant nor secure enough for production use.
+**DO NOT USE THIS SERVER IN A PRODUCTION SETTING.** The development server is for development and testing purposes only. It is neither performant nor secure enough for production use.
 
 !!! warning
-    If the test service does not run, or you cannot reach the Nautobot home page, something has gone wrong. Do not proceed with the rest of this guide until the installation has been corrected. Some platforms (such as CentOS) have a firewall enabled by default.  If you are unable to connect to the server url on port 8080, verify the firewall policy to allow the appropriate connections, or select  ...
+If the test service does not run, or you cannot reach the Nautobot home page, something has gone wrong. Do not proceed with the rest of this guide until the installation has been corrected. Some platforms (such as CentOS) have a firewall enabled by default. If you are unable to connect to the server url on port 8080, verify the firewall policy to allow the appropriate connections, or select an al ...
 
 !!! important
-    Certain Nautobot features (Git repository synchronization, webhooks, jobs, etc.) depend on the presence of Nautobot's background Celery worker process, which is not automatically started by the `runserver` command. To start it for testing purposes, you can run `nautobot-server celery worker` (for background tasks) or `nautobot-server rqworker` (for jobs) separately. For production use, Nautob ...
+Certain Nautobot features (Git repository synchronization, webhooks, jobs, etc.) depend on the presence of Nautobot's background Celery worker process, which is not automatically started by the `runserver` command. To start it for testing purposes, you can run `nautobot-server celery worker` (for background tasks) or `nautobot-server rqworker` (for jobs) separately. For production use, Nautobot a ...
````
revert

glennmatthews commented on nautobot/docs/installation/services.md:
```diff
@@ -15,19 +15,19 @@ This document will guide you through setting up uWSGI and establishing Nautobot
 
 +/- 1.1.0
-    Prior to version 1.1.0, Nautobot utilized RQ as the primary background task worker. As of Nautobot 1.1.0, RQ is now *deprecated*. RQ and the `@job` decorator for custom tasks are still supported for now, but users should [migrate the primary worker to Celery](#migrating-to-celery-from-rq) and then, *only if still required*, [run RQ concurrently with the Celery worker](#concurrent-celery-and-r ...
+Prior to version 1.1.0, Nautobot utilized RQ as the primary background task worker. As of Nautobot 1.1.0, RQ is now _deprecated_. RQ and the `@job` decorator for custom tasks are still supported for now, but users should [migrate the primary worker to Celery](#migrating-to-celery-from-rq) and then, _only if still required_, [run RQ concurrently with the Celery worker](#concurrent-celery-and-rq-na ...
```
revert

glennmatthews commented on nautobot/docs/installation/services.md:
```diff
@@ -98,14 +98,14 @@ address and/or port number, or to make performance-related adjustments. See [uWS
 
 !!! note
-    If you are deploying uWSGI behind a load balancer be sure to configure the harakiri timeout and keep alive appropriately.
+If you are deploying uWSGI behind a load balancer be sure to configure the harakiri timeout and keep alive appropriately.
```
revert

glennmatthews commented on nautobot/docs/installation/services.md:
```diff
@@ -98,14 +98,14 @@ address and/or port number, or to make performance-related adjustments. See [uWS
 
 !!! warning
-    The following steps must be performed with root permissions.
+The following steps must be performed with root permissions.
```
revert

glennmatthews commented on nautobot/docs/installation/services.md:
```diff
@@ -142,7 +142,7 @@ WantedBy=multi-user.target
 
 +/- 1.1.0
-    Prior to version 1.1.0, Nautobot utilized RQ as the primary background task worker. As of Nautobot 1.1.0, RQ is now *deprecated* and has been replaced with Celery. RQ can still be used by plugins for now, but will be removed in a future release. Please [migrate your deployment to utilize Celery as documented below](#migrating-to-celery-from-rq).
+Prior to version 1.1.0, Nautobot utilized RQ as the primary background task worker. As of Nautobot 1.1.0, RQ is now _deprecated_ and has been replaced with Celery. RQ can still be used by plugins for now, but will be removed in a future release. Please [migrate your deployment to utilize Celery as documented below](#migrating-to-celery-from-rq).
```
revert

glennmatthews commented on nautobot/docs/installation/services.md:
```diff
@@ -257,7 +257,7 @@ index f84073fb5..52baf6096 100644
 
 !!! warning
-    Failure to account for the Celery-to-RQ migration may break your custom background tasks
+Failure to account for the Celery-to-RQ migration may break your custom background tasks
```
revert

glennmatthews commented on nautobot/docs/installation/services.md:
```diff
@@ -311,7 +311,7 @@ sudo systemctl enable --now nautobot-rq-worker
 
 !!! tip
-    If you are running the concurrent RQ worker, you must remember to enable/check/restart the `nautobot-rq-worker` process as needed, oftentimes in addition to the `nautobot-worker` process.
+If you are running the concurrent RQ worker, you must remember to enable/check/restart the `nautobot-rq-worker` process as needed, oftentimes in addition to the `nautobot-worker` process.
```
revert

glennmatthews commented on nautobot/docs/installation/upgrading.md:
```diff
@@ -51,33 +56,33 @@ Please see the [official MySQL documentation on migrating collation encoding set
 
 !!! tip
-    If you wish to migrate from PostgreSQL to MySQL, we recommend creating a new Nautobot installation based on MySQL and then [migrating the database contents to the new installation](./migrating-from-postgresql.md), rather than attempting an in-place upgrade or migration.
+If you wish to migrate from PostgreSQL to MySQL, we recommend creating a new Nautobot installation based on MySQL and then [migrating the database contents to the new installation](./migrating-from-postgresql.md), rather than attempting an in-place upgrade or migration.
```
revert

glennmatthews commented on nautobot/docs/installation/upgrading.md:
```diff
@@ -51,33 +56,33 @@ Please see the [official MySQL documentation on migrating collation encoding set
 
 !!! warning
-    Unless explicitly stated, all steps requiring the use of `pip3` or `nautobot-server` in this document should be performed as the `nautobot` user!
+Unless explicitly stated, all steps requiring the use of `pip3` or `nautobot-server` in this document should be performed as the `nautobot` user!
```
revert

glennmatthews commented on nautobot/docs/models/extras/webhook.md:
```diff
@@ -4,44 +4,43 @@ A webhook is a mechanism for conveying to some external system a change that too
 
-+++ 1.3.0
-    * `snapshots` - snapshots of the serialized object state both before and after the change was made; provided as a dictionary with keys named `prechange`, `postchange` and `differences`.
++++ 1.3.0 \* `snapshots` - snapshots of the serialized object state both before and after the change was made; provided as a dictionary with keys named `prechange`, `postchange` and `differences`.
```
revert

glennmatthews reviewed (changes requested):
Looks like you let an autoformatter run amok on your documentation changes. At a *minimum* you need to revert all changes to the admonitions in these documents where the autoformatter has removed the required 4-space leading indent, as this will break the rendering of these admonitions; IMHO it would be preferable to also revert the unnecessary changes to table formatting, emphasis formatting (`*...*` vs `_..._`), and list formatting (`-` vs `*`) as those are all substantively unrelated to the content of this PR.

jvanderaa commented:
I have gone through many of the docs again:

* Reverted all `!!!` subsections to match what should be there that the autoformatter removed
* Reverted all _italics_ using underlines to astericks
* Reverted all bullets to using astericks
* Kept in minor spacing changes
  * A few sections where new lines were added where the line could be on one continuous line
  * A couple of two spaces after periods moved to single spaces

glennmatthews commented on nautobot/docs/installation/centos.md:
````diff
@@ -181,7 +197,12 @@ If successful, you will enter a `mysql>` prompt. Type `status` to confirm your c
+```no-highlight
 Enter password:
 Welcome to the MySQL monitor.  Commands end with ; or \g.
 Your MySQL connection id is 10
````
Below here, `mysql>` is a prompt.

glennmatthews commented on nautobot/docs/installation/http-server.md:
```diff
@@ -176,15 +174,20 @@ Example of correct permissions:
 [root@localhost ~]# ls -l /opt/
```
`[root@localhost ~]#` is a prompt.

glennmatthews commented on nautobot/docs/installation/migrating-from-postgresql.md:
````diff
@@ -32,7 +32,7 @@ In very rare cases, there may problems when importing your data from the case-se
 
 ```no-highlight
-(nautobot-mysql) $ nautobot-server dbshell
+nautobot-server dbshell
````
Some loss of context in the changes in this file (there are two separate Nautobot instances, one running Postgres and one running MySQL). May need to add some text before these various commands to make it clearer which instance each command is to be run against.

glennmatthews commented on nautobot/docs/installation/ubuntu.md:
```diff
@@ -42,11 +42,15 @@ At a minimum, we need to create a database for Nautobot and assign it a username
-$ sudo -u postgres psql
 psql (12.5 (Ubuntu 12.5-0ubuntu0.20.04.1))
 Type "help" for help.
 
```
Below here, `postgres=#` is a prompt.

glennmatthews commented on nautobot/docs/installation/ubuntu.md:
````diff
@@ -66,7 +70,12 @@ You can verify that authentication works issuing the following command and provi
+```no-highlight
 Password for user nautobot:
 psql (12.5 (Ubuntu 12.5-0ubuntu0.20.04.1))
 SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
````
Below here, `nautobot=>` is a prompt.

glennmatthews commented on nautobot/docs/installation/ubuntu.md:
````diff
@@ -99,7 +108,12 @@ At a minimum, we need to create a database for Nautobot and assign it a username
+```no-highlight
 Welcome to the MySQL monitor.  Commands end with ; or \g.
 Your MySQL connection id is 11
 Server version: 8.0.25-0ubuntu0.20.04.1 (Ubuntu)
````
Below here, `mysql>` is a prompt.

glennmatthews commented on nautobot/docs/installation/upgrading.md:
````diff
@@ -39,7 +39,12 @@ The collation encoding is used to inform MySQL how characters are sorted in the
+Example output:
+
+```no-highlight
 mysql> ALTER DATABASE nautobot COLLATE utf8mb4_0900_ai_ci;
````
`mysql>` is a prompt.

glennmatthews commented on nautobot/docs/plugins/development.md:
```diff
@@ -1150,7 +1160,7 @@ This view retrieves a random animal from the database and and passes it as a con
 ### Utilizing Nautobot Generic Views
 
-Starting in Nautobot 1.1.0 via [PR](https://github.com/nautobot/nautobot/issues/14), some `generic` views have been exposed to help aid in plugin development.  These views have some requirements that must be in place in order to work.  These can be used by importing them from `from nautobot.core.views import generic`.
+Starting in Nautobot 1.1.0 via [PR](https://github.com/nautobot/nautobot/issues/14), some `generic` views have been exposed to help aid in plugin development. These views have some requirements that must be in place in order to work. These can be used by importing them from `from nautobot.core.views import generic`.
```
```suggestion
+++ 1.1.0
    Via [PR #14](https://github.com/nautobot/nautobot/issues/14), some `generic` views have been exposed to help aid in plugin development. These views have some requirements that must be in place in order to work. These can be used by importing them from `from nautobot.core.views import generic`.
```

glennmatthews commented on nautobot/docs/rest-api/filtering.md:
```diff
@@ -9,17 +9,22 @@ Multiple parameters can be joined to further narrow results. For example, `GET /
 
 +/- 1.4.0
-    If [STRICT_FILTERING](../configuration/optional-settings.md#strict_filtering) is True (its default value), unrecognized filter parameters now result in a 400 Bad Request response instead of being silently ignored.
+If [STRICT_FILTERING](../configuration/optional-settings.md#strict_filtering) is True (its default value), unrecognized filter parameters now result in a 400 Bad Request response instead of being silently ignored.
```
```suggestion
    If [STRICT_FILTERING](../configuration/optional-settings.md#strict_filtering) is True (its default value), unrecognized filter parameters now result in a 400 Bad Request response instead of being silently ignored.
```

glennmatthews commented on nautobot/docs/rest-api/filtering.md:
```diff
@@ -41,7 +46,7 @@ http://nautobot/api/ipam/prefixes/ | jq ".actions.POST.status.choices"
 
 !!! note
-    The above works only if the API token used to authenticate the request has permission to make a `POST` request to this endpoint.
+The above works only if the API token used to authenticate the request has permission to make a `POST` request to this endpoint.
```
```suggestion
    The above works only if the API token used to authenticate the request has permission to make a `POST` request to this endpoint.
```

glennmatthews commented on nautobot/docs/rest-api/filtering.md:
```diff
@@ -52,12 +57,12 @@ GET /api/dcim/sites/?cf_foo=123
 
 !!! note
-    For custom field filters, due to historical details of implementation, only a single filter value can be specified when matching a given field. In other words, in the above example, you could _not_ add `&cf_foo=456` to the query in order to get all sites where custom field `foo` is 123 _or_ 456; instead you would need to run two separate queries. This restriction does not apply to custom fiel ...
+For custom field filters, due to historical details of implementation, only a single filter value can be specified when matching a given field. In other words, in the above example, you could _not_ add `&cf_foo=456` to the query in order to get all sites where custom field `foo` is 123 _or_ 456; instead you would need to run two separate queries. This restriction does not apply to custom field fi ...
```
```suggestion
    For custom field filters, due to historical details of implementation, only a single filter value can be specified when matching a given field. In other words, in the above example, you could _not_ add `&cf_foo=456` to the query in order to get all sites where custom field `foo` is 123 _or_ 456; instead you would need to run two separate queries. This restriction does not apply to custom field filters using lookup expressions (next section) and will likely be changed in a future major version of Nautobot.
```

glennmatthews commented on nautobot/docs/rest-api/filtering.md:
```diff
@@ -52,12 +57,12 @@ GET /api/dcim/sites/?cf_foo=123
 
 +++ 1.4.0
-    Custom fields can use the lookup expressions listed in the next section by prepending `cf_` to the custom field `name` (and not the `slug`) followed by the required lookup type (see below).
+Custom fields can use the lookup expressions listed in the next section by prepending `cf_` to the custom field `name` (and not the `slug`) followed by the required lookup type (see below).
```
```suggestion
    Custom fields can use the lookup expressions listed in the next section by prepending `cf_` to the custom field `name` (and not the `slug`) followed by the required lookup type (see below).
```

glennmatthews commented on nautobot/docs/rest-api/filtering.md:
```diff
@@ -90,11 +95,7 @@ String-based (char) fields (Name, Address, etc.) support these lookup expression
 - `ie` - case-insensitive exact match
 - `nie` - negated case-insensitive exact match
 
-+++ 1.3.0
-    - `re` - case-sensitive regular expression match
-    - `nre` - negated case-sensitive regular expression match
-    - `ire` - case-insensitive regular expression match
-    - `nire` - negated case-insensitive regular expression match
++++ 1.3.0 - `re` - case-sensitive regular expression match - `nre` - negated case-sensitive regular expression match - `ire` - case-insensitive regular expression match - `nire` - negated case-insensitive regular expression match
```
revert please

glennmatthews reviewed (commented):
Thanks for the cleanup!

jvanderaa commented on nautobot/docs/development/getting-started.md:
````diff
@@ -278,10 +313,20 @@ This will create automatically create a virtualenv in your home directory, which
+
+```no-highlight
 Spawning shell within /home/example/.cache/pypoetry/virtualenvs/nautobot-Ams_xyDt-py3.8
 
-$ . /home/example/.cache/pypoetry/virtualenvs/nautobot-Ams_xyDt-py3.8/bin/activate
+. /home/example/.cache/pypoetry/virtualenvs/nautobot-Ams_xyDt-py3.8/bin/activate
+```
+
+Example output:
+
+```no-highlight
 (nautobot-Ams_xyDt-py3.8) $
 ```
````
With a fresh mind I'm agreeing. Not sure what my eyes were seeing at that moment.

jvanderaa commented on nautobot/docs/installation/centos.md:
````diff
@@ -82,14 +82,20 @@ At a minimum, we need to create a database for Nautobot and assign it a username
+```
 
+```no-highlight
 postgres=# CREATE DATABASE nautobot;
````
I'm hesitent at the moment on the SQL output. In this case at the moment it is nice to have it wrapped up in one place for SQL. I would like to add my other documentation idea of tabbed sections and address it there?

glennmatthews replied:
That's fine by me.

jvanderaa commented on nautobot/docs/docker/index.md:
````diff
@@ -232,9 +244,23 @@ nautobot:
+Example output:
+
+```no-highlight
 ...
````
Agree

glennmatthews commented on nautobot/docs/installation/http-server.md:
````diff
@@ -170,21 +168,31 @@ If you are unable to connect to the HTTP server, check that:
+```no-highlight
 total 4
 drwx------. 11 nautobot nautobot 4096 Apr  5 10:00 nautobot
 [nautobot@localhost ~]$ chmod 755 $NAUTOBOT
````
Couple of missed prompts here

glennmatthews commented on nautobot/docs/installation/nautobot.md:
````diff
@@ -126,15 +146,21 @@ Before we install anything into the virtualenv, we want to make sure that Pip is
-$ pip3 install --upgrade pip wheel
+pip3 install --upgrade pip wheel
+```
+
+Example output:
+
+```no-highlight
+
 ```
````
````suggestion
````

glennmatthews commented on nautobot/docs/rest-api/filtering.md:
````diff
@@ -16,10 +16,15 @@ Generally, passing multiple values for a single parameter will result in a logic
+
+Example output:
+
+```no-highlight
````
```suggestion
```json
```

glennmatthews reviewed (approved):
A few remaining very minor nits. Thanks for the work here!
